src/strategies/esd/index.ts

```diff
@@ -0,0 +1,97 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'lewi';
```

Comment:
This must be your GitHub username
```suggestion
export const author = 'l3wi';
```